public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -45,7 +49,7 @@ export function createSpanLinkFactory({
           field,
           rowIndex: span.dataFrameRowIndex!,
           splitOpenFn,
-          range: getTimeRangeFromSpan(span),
+          range: getTimeRangeFromSpan(span, isSplunkDS),
```

Comment:
Trace to logs doesn't go through this SpanLink function, so we can remove this argument (maybe give this parameter a default or create two getTimeRangeFromSpan functions - one for Loki and one for Splunk)